overseerr-api.yml

```diff
@@ -1039,6 +1039,8 @@ components:
           nullable: true
         status:
           type: number
+          example: 0
+          description: Availability of the media. 1 = UNKNOWN, 2 = PENDING, 3 = PROCESSING, 4 = PARTIAL, 5 = AVAILABLE
```

Comment:
```suggestion
          description: Availability of the media. 1 = `UNKNOWN`, 2 = `PENDING`, 3 = `PROCESSING`, 4 = `PARTIALLY_AVAILABLE`, 5 = `AVAILABLE`
```

Reply:
@ceptonit If we can get this suggestion also added we can merge this 😄